Snake/BusinessLayer/Engine.cs

```diff
@@ -6,6 +6,11 @@
 
 namespace BusinessLayer
 {
+    public enum direction
+    {
+        right, down, left, up
+    }
+
     public class Engine
     {
         private const int SNAKEINITIALLENGTH = 4;
```

Comment:
don't use upper case for constants. just follow the same convention as you would a variable. in this case that would be camel because it is private